src/polysolve/LinearSolver.hpp

```diff
@@ -85,6 +85,14 @@ namespace polysolve
         // Factorize system matrix
         virtual void factorize(const StiffnessMatrix &A) {}
 
+        #ifdef POLYSOLVE_WITH_CUSOLVER
```

Comment:
no ifdef, we can just have overload for the dense solvers

Reply:
removed ifdef section